### PR TITLE
fix: fixed namespace webhooks

### PR DESCRIPTION
**Pull request by sagar-jadhav**
Fixes https://github.com/clastix/capsule/issues/733

### CODE REVIEW

File: pkg/webhook/namespace/patch.go [hunk 1]
```diff
@@ -72,11 +72,6 @@ func (r *patchHandler) OnUpdate(c client.Client, decoder *admission.Decoder, rec
 
 				return &response
 			}
-		} else {
-			recorder.Eventf(ns, corev1.EventTypeWarning, "NamespacePatch", e)
-			response := admission.Denied(e)
-
-			return &response
```
Comment (prometherion) [inline pkg/webhook/namespace/patch.go:79]: I'm a bit concerned about removing this, may I ask you for some support in getting the reason?

File: pkg/webhook/namespace/owner_reference.go [hunk 2]
```diff
@@ -42,6 +42,10 @@ func (r *ownerReferenceHandler) OnUpdate(_ client.Client, decoder *admission.Dec
 			return utils.ErroredResponse(err)
 		}
 
+		if len(oldNs.OwnerReferences) == 0 {
+			return nil
+		}
+
```
Comment (prometherion) [inline pkg/webhook/namespace/owner_reference.go:48]: This is a nice catch!

File: pkg/webhook/namespace/patch.go (hunk 1 above)
Comment (rakshpatel) [reply pkg/webhook/namespace/patch.go:79]: @prometherion As @sagar-jadhav stated above non-capsule managed namespaces will not have this label, in that case, this else will always block. Is there any use case for which this is needed?

Comment (prometherion) [reply pkg/webhook/namespace/patch.go:79]: You're right, thanks for pointing it out!

**Pull request merged**